doc/source/free-distribution-testing.md

```diff
@@ -0,0 +1,437 @@
+
+![worker](_static/images/poisson-distribution/worker.jpg)
+
+|Before (1)|After(2)|
+|:----:|:----:|
+|8|6|
+|7|5|
+|6|8|
+|9|6|
+|7|9|
+|10|8|
+|8|10|
+|6|7|
+|5|5|
+|8|6|
+|10|9|
+|8|8|
+
+The hypotheses are as:
+
+`Ho: Me1 ≤ Me2`
+`Ha: Me1 > Me2`
+
+The first thing we need to do is subtract the values between the defective units before the salary increase and the number of defective items after the incentive. To then order the absolute value of the range from lowest to highest, assigning values from 1 onwards.
+
+|Before (X1)|After (X2)|X1 – X2|Sign|Rank|
+|:---:|:---:|:---:|:---:|:---:|
+|8|6|2|+|6|
+|7|5|2|+|6|
+|6|8|-2|-|6|
+|9|6|3|+|10|
+|7|9|2|+|6|
+|10|8|2|+|6|
+|8|10|-2|-|6|
+|6|7|-1|-|1.5|
+|5|5|0||-|
+|8|6|-2|-|6|
+|10|9|1|+|1.5|
+|8|8|0||-|
+
+As we can see, the absolute value of 1 is repeated twice and that of 2 is repeated seven times. So the absolute minimum value (1) occupies places 1 and 2, so the average between the two is 1.5. In the same way, the absolute value of 2 occupies places 3, 4, 5, 6, 7, 8 and 9, so its average is: (3 + 4 + 5 + 6 + 7 + 8 + 9) / 7 = 6. It is important to emphasize that the zeros (0) are not taken into account neither for the range nor for the number of observations, so in this case n = 10 instead of 12 as we had initially.
+
+Adding the range of the positive and negative numbers we have:
+
+`T (+) = 35.5`
+`T (-) = 19.5`
+
+As the alternative hypothesis tells us that the mean value must be “Greater than”, we assume that the Tcal is the greater of both values. If we go back to the tables above and look at the hypotheses, we can conclude that this is a right-tailed test.
+
+`Tcal = 35.5`
+
+We can then say that the null hypothesis is rejected, when the alternative hypothesis is true and this occurs when `Tcal > T (α, n).`
+
+To find `T (α, n)` we go to the Table of critical values of the Wilcoxon T test for a single tail, with a significance of 0.01 and a value of n = 10
+
+![chart6](_static/images/poisson-distribution/chart 6.jpg)
+
+By tables we obtain that T (α, n) = 5, if we compare this value with that obtained from the Tcal we have:
+
+`35.5 > 5`
+
+The alternative hypothesis is then proved and therefore the null hypothesis is rejected. In this way we conclude that Me1 > Me2, which implies that the median of the defective production after the wage incentive, is definitely less than the median before the increase. This means that the task of lowering the production of defective units was achieved by raising the salary of its employees.
+
+## About the Author
+
+Graduated in Mechanical Engineering, and a master’s degree in teaching component, I gave classes in several institutes of mathematics and physics, but I also dedicated several years of my life as a television producer, I did the scripts for mikes, the camera direction, editing of video and even the location. Later I was dedicated to SEO writing for a couple of years. I like poetry, chess and dominoes.
+
+
```

Comment:
Too many extra lines

Reply:
Sorry, about that. I have resolved it on my latest commit **Removed White Space from File**

Reply:
sorry about that,  resolved it in my recent commit.